### ISSUE TITLE
What happens if I set padding = -1 ? 

### ISSUE THREAD

**Issue by bboymoonchild** [pytorch/pytorch#39013]
Hi. I am studying Conv. Network at torch.nn 

I saw some code that 

nn.Conv2d(cin, cout, kernel_size=k, stride=stride, **padding=-1**, bias=False)

Since I was using keras before, I could not find what happens when padding is -1. 

Can anybody tell me what happens by the value of padding ?

**Comment by ezyang** [comment]
Please ask questions at https://discuss.pytorch.org/
